x-pack/qa/third-party/active-directory/src/test/java/org/elasticsearch/xpack/security/authc/ldap/AbstractActiveDirectoryTestCase.java

```diff
@@ -154,7 +154,7 @@ private static String getFromEnv(String envVar, String defaultValue) {
     }
 
     private static String getFromProperty(String port) {
-        String key = "test.fixtures.smb-fixture.fixture." + port;
+        String key = "test.fixtures.smb-fixture.fixture.tcp." + port;
```

Comment:
nit: why is this key so long? The last "fixture" seems redundant?

Reply:
the second "fixture" is the name of the service from the `docker-compose.yml`. We could rename that to something else, it's needed because there can be multiple services for one docker-compose file.

Reply:
Why not have "smb-fixture" be the name of the service in docker-compose? ie:
`test.fixtures.smb-fixture.tcp`